cmd/obslytics/export.go

```diff
@@ -2,41 +2,95 @@ package main
 
 import (
 	"context"
+	"fmt"
+	"os"
+	"time"
 
 	"github.com/go-kit/kit/log"
 	"github.com/oklog/run"
-	infactory "github.com/thanos-community/obslytics/pkg/input/factory"
-	outfactory "github.com/thanos-community/obslytics/pkg/output/factory"
+	"github.com/thanos-community/obslytics/pkg/input"
+	"github.com/thanos-community/obslytics/pkg/output"
+
+	"github.com/thanos-community/obslytics/pkg/output/debug"
 	"github.com/thanos-io/thanos/pkg/extflag"
 	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/thanos-community/obslytics/pkg/ingest"
+	infactory "github.com/thanos-community/obslytics/pkg/input/factory"
+	outfactory "github.com/thanos-community/obslytics/pkg/output/factory"
 )
 
 func registerExport(m map[string]setupFunc, app *kingpin.Application) {
 	cmd := app.Command("export", "Export Observability Data into popular analytics formats.")
-	inputFlag := extflag.RegisterPathOrContent(cmd, "input", "YAML for input configuration.", true)
-	outputFlag := extflag.RegisterPathOrContent(cmd, "ouput", "YAML for output configuration.", true)
+	inputFlag := extflag.RegisterPathOrContent(cmd, "input-config", "YAML for input configuration.", true)
+	// TODO(inecas): add support for output configuration.
+	outputFlag := extflag.RegisterPathOrContent(cmd, "output-config", "YAML for output configuration.", false)
+
+	seriesParams := input.SeriesParams{}
+	cmd.Flag("match", "Name of the metric to export").Required().StringVar(&seriesParams.Matcher)
+	timeFmt := time.RFC3339
+
+	cmd.Flag("min-time", fmt.Sprintf("The lower boundary of the time series in %s or duration format", timeFmt)).
+		Required().SetValue(&seriesParams.MinTime)
+
+	cmd.Flag("max-time", fmt.Sprintf("The upper boundary of the time series in %s or duration format", timeFmt)).
+		Required().SetValue(&seriesParams.MaxTime)
+
+	var resolution time.Duration
```

Comment:
BTW all of this is not needed if we do `resolution := cmd.Flag("resolution", "Sample resolution (e.g. 30m)").Required().Duration()`

Reply:
but not a blocker for now